internal/receiver/databricksreceiver/testdata/clusters-list.json

```diff
@@ -0,0 +1,140 @@
+{
+  "clusters": [
+    {
+      "cluster_id": "1110-200233-w85kg6q",
+      "creator_user_name": "mvolker@splunk.com",
```

Comment:
maybe worth anonymizing the email throughout this file if possible

Reply:
Thanks, will be making several changes before this PR is ready for review.